pages/vi/profiles/hemu-codes.md

```diff
@@ -0,0 +1,25 @@
+# Hemantha 
+
+
+-----------------------------
+
+* Full Name: Hemantha Akkaraju
+* Location: Seattle, Washington
+* Time Zone: PST
+* OS: Mac Version 10.14.5
+-------------------------------------
+
+## Description: 
+Hello, I am Hemantha - a sophomore attending the University of Washington Bothell. 
+I am pursuing a degree in Mathematics and Computer Science. Some of the extracurricular activities I am part of include 
+Undergraduate Research and officer for some of the organizations at my school. I am really excited to be part of this team
+and looking forward to working with you all!!
+
+[Linkedin](www.linkedin.com/in/hemantha-akkaraju)
```

Comment:
link does not work

Reply:
Fixed the link! Could you please check if it works now? Thanks again!